README.md

```diff
@@ -11,45 +11,52 @@ The objective of the morning is:
-- You can find it here: https://github.com/corndeladmin/devops-pub-quiz
-- Fork it by pressing the "Fork" button. 
-    - You can read more about forking on various resources, including [How to Fork a GitHub Repository – A Complete Workflow](https://www.freecodecamp.org/news/how-to-fork-a-github-repository/)
-- Clone the repo to your machine by following these instructions:
-    - Go to the main page of the repository you want to clone on GitHub.com (in this case, just follow the link above)
-    - Click on the green "Code" button. A small window will pop up showing the repository URL.
-    - Copy the repository URL. You can click the copy icon next to the URL to copy it to your clipboard.
-    - Open your terminal (Mac/Linux, including ACG) or Git Bash (Windows).
-    - Navigate to the directory where you want the cloned repository to be saved using the cd command. For example: `cd /path/to/your/directory`
-    - Run the `git clone` command followed by the URL you copied. For example: `git clone https://github.com/username/repository.git`
-    - Press Enter. Git will start cloning the repository to your local machine.
-- Once it's downloaded, open it in Visual Studio Code.
-
-### Doing the work
-- In VS Code, read the existing application and understand how it works.
-- Run it by doing `python app.py`
-- lt should run, and you should experience a simple quiz with 2 sample questions!
-- Now you’re satisfied that the code works and you understand at least at some level how it works, let’s modify it.
-- Your goal for this is to improve the Pub Quiz experience for the user
-    - The simplest way to do this is simply to add a single question!
-    - If you’ve done that, you should expand your efforts:
-        - Replace the existing questions with ones of your own!
-        - You can give the user a score at the end by keeping track of it
-        - You could create non-multiple choice questions 
-        - Or any other change you are inspired to make!
-
-### Doing the workflow
-- Once you’ve made your first change, you should commit it to Git and push it to GitHub. 
-    - Check that you are able to make a PR - read and follow these instructions: [Creating a pull request - GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
-- Roughly an hour to half an hour before lunch we’ll get everyone to give each other feedback.
-    - At this point, you should update your PR to contain all your latest changes, and provide a link to the PR to your PDE. 
-    - Your PDE will pass your PR to someone else, and pass you a link to someone else’s PR.
-    - Download their code and run it on your machine. GitHub shows you how to do it here: [GitHub Guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally)
```

Comment:
This doesn't work as you need to have cloned the original repo and not a fork of said original repo. Easier just to clone the repo a learner's been asked to review.